changelog/fix-release-code-freeze-workflow-secret

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+
```

Comment:
Could we add something in the changelog for reference?

Reply:
Ah. I meant to add a non significant entry (so it doesn't show up in the actual changelog generated), but forgot to put any text. Will fix it